server/package.json

```diff
@@ -15,6 +15,7 @@
 	"author": "JanPaul Bergeson",
 	"license": "MIT",
 	"devDependencies": {
+		"@types/jest": "^29.5.12",
```

Comment:
?

Reply:
The problem is that if you try to compile the ts scripts with npm run compile, to get a lot of errors related with the tests:

```
src/parser/test/formatter.test.ts:633:5 - error TS2582: Cannot find name 'it'. Do you need to install type definitions for a test runner? Try `npm i --save-dev @types/jest` or `npm i --save-dev @types/mocha`.

633     it('preserves comments after statements', () => {

```

this is why I added the dependency, so I could compile and continue with the process. Maybe there is some alternative to compile without the tests? but I think that it would be required to touch the package.json anyway.